include/extractor/turn_instructions.hpp

```diff
@@ -31,37 +37,87 @@ enum class TurnInstruction : unsigned char
     AccessRestrictionPenalty = 129
 };
 
+// shiftable turns to left and right
+const constexpr bool shiftable_left[] = {false, false, true, true, true, false, false, true, true};
+const constexpr bool shiftable_right[] = {false, false, true, true, false, false, true, true, true};
```

Comment:
Add static assertion for the size of the array `w.r.t` to the number of turn instructions. You can extend the enum to include `NumberOfTurnInstructions` to get the size automatically.